Data/Sector/Hong-Kong-Sector-File.ese

```diff
@@ -2453,14 +1982,179 @@ COORD:N023.30.00.000:E117.30.00.000
 DISPLAY:VHHK·VHHK_SCT_TRE_01·000·250:VHHK·VHHK_SCT_TRE_01·000·250:ZSHA·ZSAM·000·256
 
+SECTORLINE:106 ;TRW North
+COORD:N022.07.06.546:E113.39.52.148
+COORD:N022.09.34.169:E113.41.53.138
+COORD:N022.21.29.999:E113.52.00.001
+COORD:N022.23.53.542:E113.52.00.001
+COORD:N022.26.16.884:E113.54.31.218
+COORD:N022.32.47.326:E114.12.41.016
+COORD:N022.32.27.490:E114.13.07.872
+COORD:N022.34.50.000:E114.16.05.000
+DISPLAY:VHHK·VHHK_SCT_TRW_01·250·365:VHHK·VHHK_SCT_TRW_01·250·365:ZGZU·ZGGG·256·660
+
+SECTORLINE:107 ;(Visual only) 07 DEH PRC handoff
+COORD:N022.19.59.852:E113.45.36.426
+COORD:N022.24.34.000:E113.49.44.000 
+COORD:N022.26.16.884:E113.54.31.218
+COORD:N022.32.47.326:E114.12.41.016
+COORD:N022.32.27.490:E114.13.07.872
+COORD:N022.34.50.000:E114.16.05.000
+COORD:N022.34.47.978:E114.18.13.952
+COORD:N022.36.32.635:E114.23.53.000
+COORD:N022.32.27.709:E114.28.00.598
+COORD:N022.27.43.342:E114.29.03.948
+COORD:N022.26.56.963:E114.30.08.265
+COORD:N022.23.50.616:E114.30.47.871
+COORD:N022.28.17.000:E114.41.57.470
+DISPLAY:VHHK·VHHK_SCT_07_DEH_04·165·250:VHHK·VHHK_SCT_07_DEH_04·165·250:ZGZU·ZGGG·000·256
+DISPLAY:VHHK·VHHK_SCT_07_DEP_02·000·165:VHHK·VHHK_SCT_07_DEP_02·000·165:ZGZU·ZGGG·000·256
+DISPLAY:VHHK·VHHK_SCT_07_DEP_02·000·165:VHHK·VHHK_SCT_07_DEP_02·000·165:ZGZU·ZGJD_APP·000·148
+DISPLAY:VHHK·VHHK_SCT_07NAP_DEH_04·165·250:VHHK·VHHK_SCT_07NAP_DEH_04·165·250:ZGZU·ZGGG·000·256
+DISPLAY:VHHK·VHHK_SCT_07NAP_DEH_03·135·250:VHHK·VHHK_SCT_07NAP_DEH_03·135·250:ZGZU·ZGGG·000·256
+
+SECTORLINE:107a ;(Visual only) 07 FAD PRC handoff
+COORD:N022.07.06.546:E113.39.52.148
+COORD:N022.09.34.169:E113.41.53.138
+COORD:N022.13.44.000:E113.39.57.000
+COORD:N022.19.59.852:E113.45.36.426
+DISPLAY:VHHK·VHHK_SCT_07_FAD_01·000·100:VHHK·VHHK_SCT_07_FAD_01·000·100:ZGZU·ZGGG·000·256
+DISPLAY:VHHK·VHHK_SCT_07_FAD_01·000·100:VHHK·VHHK_SCT_07_FAD_01·000·100:ZGZU·ZGJD_APP·000·148
+DISPLAY:VHHK·VHHK_SCT_07NAP_DEH_01·100·250:VHHK·VHHK_SCT_07NAP_DEH_01·100·250:ZGZU·ZGGG·000·256
+DISPLAY:VHHK·VHHK_SCT_07NAP_DEH_01·100·250:VHHK·VHHK_SCT_07NAP_DEH_01·100·250:ZGZU·ZGJD_APP·000·148
+DISPLAY:VHHK·VHHK_SCT_07_DEH_01·100·250:VHHK·VHHK_SCT_07_DEH_01·100·250:ZGZU·ZGGG·000·256
+DISPLAY:VHHK·VHHK_SCT_07_DEH_01·100·250:VHHK·VHHK_SCT_07_DEH_01·100·250:ZGZU·ZGJD_APP·000·148
+
+SECTORLINE:108 ;(Visual only) 25 FAD PRC handoff
+COORD:N022.24.34.000:E113.49.44.000 
+COORD:N022.26.16.884:E113.54.31.218
+COORD:N022.32.47.326:E114.12.41.016
+COORD:N022.32.27.490:E114.13.07.872
+COORD:N022.34.50.000:E114.16.05.000
+COORD:N022.34.47.978:E114.18.13.952
+COORD:N022.36.32.635:E114.23.53.000
+COORD:N022.32.27.709:E114.28.00.598
+COORD:N022.27.43.342:E114.29.03.948
+COORD:N022.26.56.963:E114.30.08.265
+COORD:N022.23.50.616:E114.30.47.871
+COORD:N022.28.17.000:E114.41.57.470
+DISPLAY:VHHK·VHHK_SCT_25_FAD_01·000·085:VHHK·VHHK_SCT_25_FAD_01·000·085:ZGZU·ZGGG·000·256
+DISPLAY:VHHK·VHHK_SCT_25_DEH_01·135·250:VHHK·VHHK_SCT_25_DEH_01·135·250:ZGZU·ZGGG·000·256
```

Comment:
Extra line here
Config below: on VHHH_F_APP, other controllers online shown
![image](https://github.com/vatsimhk/Hong-Kong-Sector-Package/assets/55276992/64e6919a-98e5-404b-9036-ceca63ea7341)

Also, shouldn't this also display if Zhuhai APP is online? (BEKOL transfer)